user_guide.adoc

```diff
@@ -1377,30 +1377,82 @@ library" for expression language.
 |Return type
 |Description
 
-|to_s
+|`to_s`
 |String
 |Converts integer into a string using decimal representation
-
-|to_s(radix)
```

Comment:
This is [exactly it](https://github.com/kaitai-io/kaitai_struct_compiler/blob/add79599cc7d0a471b1908f4028ff0080a872f56/shared/src/main/scala/io/kaitai/struct/translators/BaseTranslator.scala#L117) - it accepts one radix argument here.

There's also [an argument-less version](https://github.com/kaitai-io/kaitai_struct_compiler/blob/add79599cc7d0a471b1908f4028ff0080a872f56/shared/src/main/scala/io/kaitai/struct/translators/BaseTranslator.scala#L85) in slightly different place.